data/reusables/copilot/enabling-copilot-chat-beta.md

```diff
@@ -8,7 +8,7 @@
 
 {% data variables.product.prodname_copilot_chat %} is available to all organizations and enterprises that have an active {% data variables.product.prodname_copilot_for_business %} license. You can enable or disable {% data variables.product.prodname_copilot_chat %} for your organization or enterprise in the {% data variables.product.prodname_copilot_for_business %} settings page.
 
-If {% data variables.product.prodname_copilot_chat_short %} is enbaled or disabled at the enterprise level, organizations within the enterprise cannot override the setting.
+If {% data variables.product.prodname_copilot_chat_short %} is enabled or disabled at the enterprise level, organizations within the enterprise cannot override the setting.
```

Comment:
#### Before
https://docs.github.com/en/enterprise-cloud@latest/copilot/github-copilot-chat/using-github-copilot-chat-in-your-ide#enabling-or-disabling-github-copilot-chat
<img width="703" alt="image" src="https://github.com/github/docs/assets/2119212/c5240ca5-a0d4-4add-8a0d-7b5f404659a8">

#### After
https://docs-30890-94eb78.preview.ghdocs.com/en/enterprise-cloud@latest/copilot/github-copilot-chat/using-github-copilot-chat-in-your-ide#enabling-or-disabling-github-copilot-chat
<img width="703" alt="image" src="https://github.com/github/docs/assets/2119212/0bf8efc0-a1e0-4ca7-ab44-af86e26ddfe0">